test/commands/scanner/run.test.ts

```diff
@@ -693,164 +693,6 @@ describe('scanner:run', function () {
 		});
 
 		describe('Dynamic Input', () => {
-			describe('Positive Globs', () => {
```

Comment:
Lines 468-579 of RuleManager.test.ts aren't an *exact* match, but they cover the same scenarios outlined here, and test them in a much more targeted way, since the primary place where Globs actually matter is in `unpackTargets()`.